main.py

```diff
@@ -2,6 +2,34 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    genres = [
+        "Western",
+        "Action",
+        "Dramma",
```

Comment:
There is a typo in the genre name 'Dramma'. It should be 'Drama'. This is corrected later in the code, but it's better to avoid such typos initially to prevent potential confusion.